src/label_studio_sdk/converter/brush.py

```diff
@@ -129,37 +129,17 @@ def decode_from_annotation(from_name, results):
 
 
 def save_brush_images_from_annotation(
-    task_id,
-    annotation_id,
-    completed_by,
-    from_name,
+    image_name,
     results,
     out_dir,
     out_format="numpy",
 ):
-    layers = decode_from_annotation(from_name, results)
-    if isinstance(completed_by, dict):
-        email = completed_by.get("email", "")
-    else:
-        email = str(completed_by)
-    email = "".join(
-        x for x in email if x.isalnum() or x == "@" or x == "."
-    )  # sanitize filename
+    layers = decode_from_annotation(results)
+    image_base = ".".join(image_name.split('.')[0:-1])
```

Comment:
if you have multiple annotations per one tasks - you will export only the last one, others will be overwritten.

Reply:
I have used the PR of label-studio-sdk to label many images that contain multiple annotations per task and this has worked through hundreds of images.  By the way, the segment anything integration with label-studio is a game changer when it comes to efficiently labeling images in a production environment.

To verify merges haven't broken anything, I pulled the latest label-studio and label-studio-sdk from the PR repo and verified with all updates functionality is preserved.   I just created a simple project to demonstrate to avoid sharing any proprietary images (see attached).   Inside the "decode_from_annotation" function there's an included sequence counter that is incremented and appended to the filename at the end of this function in the "counters[name]".  This creates a unique filename for multiple annotations in a single task.

Example export image in the "images" folder of an export:
![c2825893-NathonFamous](https://github.com/user-attachments/assets/d081f940-3848-4ac5-b267-5e117efb11af)

Example export masks in the "masks" folder of an export (each mask filename has a unique sequence name per class):
![c2825893-NathonFamous-Hotdog-0](https://github.com/user-attachments/assets/9bda2337-8812-4498-bf10-a544b674731a)
![c2825893-NathonFamous-Hotdog-1](https://github.com/user-attachments/assets/a00c303c-f24d-48b1-8506-9fc49d9ef2c4)
![c2825893-NathonFamous-Hotdog-2](https://github.com/user-attachments/assets/dc978640-8c12-44be-a48d-070ee2346150)
![c2825893-NathonFamous-Not Hotdog-0](https://github.com/user-attachments/assets/45a633ce-f9c2-44f6-a08f-55018fdfefcd)
![c2825893-NathonFamous-Not Hotdog-1](https://github.com/user-attachments/assets/b5194f49-fb34-4c41-bbfa-5c6f89d389a4)
![c2825893-NathonFamous-Not Hotdog-2](https://github.com/user-attachments/assets/639cc170-85d3-4c0e-94ba-f9c1f77e56d4)
![c2825893-NathonFamous-Not Hotdog-3](https://github.com/user-attachments/assets/70f7a409-2181-420c-a4bb-080878427526)